src/main/webapp/app/entities/outcome/outcome.component.html

```diff
@@ -1,74 +1,26 @@
-<div>
-    <h2 id="page-heading" data-cy="OutcomeHeading">
-        <span jhiTranslate="supportivecareApp.outcome.home.title">Outcomes</span>
-
-        <div class="d-flex justify-content-end">
-            <button class="btn btn-info mr-2" (click)="handleSyncList()" [disabled]="isLoading">
-                <fa-icon icon="sync" [spin]="isLoading"></fa-icon>
-                <span jhiTranslate="supportivecareApp.outcome.home.refreshListLabel">Refresh List</span>
-            </button>
-
-            <button *customHasAnyAuthority="'ROLE_USER'" id="jh-create-entity" data-cy="entityCreateButton" class="btn btn-primary jh-create-entity create-outcome" [routerLink]="['/outcome/new']">
-                <fa-icon icon="plus"></fa-icon>
-                <span  jhiTranslate="supportivecareApp.outcome.home.createLabel">
-                Create a new Outcome
-                </span>
-            </button>
-        </div>
-    </h2>
-
+<div class="mainWrapper">
+    <div class="listHeaderWrapper">
+        <h2 class="h2-color" id="page-heading" data-cy="OutcomeHeading" jhiTranslate="supportivecareApp.outcome.home.title"></h2>
+        <button *customHasAnyAuthority="'ROLE_USER'" id="jh-create-entity" data-cy="entityCreateButton" class="btn btn-primary jh-create-entity button-create" [routerLink]="['/outcome/new']">
+            <fa-icon icon="file-medical-alt" slot="start"></fa-icon>
+            <span  jhiTranslate="supportivecareApp.outcome.home.createLabel"></span>
+        </button>
+    </div>
     <custom-alert-error></custom-alert-error>
-
     <custom-alert></custom-alert>
-
     <div class="alert alert-warning" id="no-result" *ngIf="outcomes?.length === 0">
-        <span jhiTranslate="supportivecareApp.outcome.home.notFound">No outcomes found</span>
+        <span jhiTranslate="supportivecareApp.drug.home.notFound">No drugs found</span>
```

Comment:
Should be  <span jhiTranslate="supportivecareApp.outcome.home.notFound">No outcomes found</span>

Reply:
Actually, it should be "No PROMs found" since PROM is in english (and we should keep it in portuguese with the same naming)

Reply:
Also, when you use `jhiTranslate` you don't need to place a string in the `<span>` as it will override with the translation.